README.md

```diff
@@ -3,35 +3,57 @@
   This software is licensed under the Apache License version 2.
 -->
 
-# Gravitino
+![Gravitino Logo](docs/assets/gravitino-logo.png)
 
 [![GitHub Actions Build](https://github.com/datastrato/gravitino/actions/workflows/build.yml/badge.svg)](https://github.com/datastrato/gravitino/actions/workflows/build.yml)
 [![GitHub Actions Integration Test](https://github.com/datastrato/gravitino/actions/workflows/integration-test.yml/badge.svg)](https://github.com/datastrato/gravitino/actions/workflows/integration-test.yml)
 
 ## Introduction
 
-Gravitino is a high-performance, geo-distributed, and federated metadata lake. It manages the metadata directly in different sources, types, and regions. It also provides users with unified metadata access for data and AI assets.
+Gravitino is a high-performance, geo-distributed, and federated metadata lake. It creates and manages metadata 
+from multiple different database sources and cloud providers directly, regardless of type and region. Users can then 
+access the unified metadata using supported engines for data management and advanced analytics.
+
+For a detailed overview of features, please see the [official documentation](https://datastrato.ai/docs).
 
 ![Gravitino Architecture](docs/assets/gravitino-architecture.png)
 
-Gravitino aims to provide several key features:
+<details>
+<summary> Table of Contents </summary>   
```

Comment:
Use sentence case for consistency

Reply:
> Did you mean in Table of Contents?

I think "sentence case" means, only the first word use capital case: "Table of contents", instead of "Table of Contents"